src/sql/workbench/api/node/extHostModelViewDialog.ts

```diff
@@ -263,6 +301,14 @@ class WizardImpl implements sqlops.window.modelviewdialog.Wizard {
 		this._extHostModelViewDialog.registerWizardPageInfoChangedCallback(this, info => this.handlePageInfoChanged(info));
 		this._currentPage = 0;
 		this.onPageChanged(info => this._currentPage = info.newPage);
+		this._operationHandler = new BackgroundOperationHandler('wizard' + this._currentPage, extHostTaskManagement);
```

Comment:
CurrentPage will always be 0 here. Same sort of thing as my comment above about dialogs, does it need to be unique?

Reply:
there's a unique id is added to this name so the result task id will be unique 

---
In reply to: [202498480](https://github.com/Microsoft/sqlopsstudio/pull/1929#discussion_r202498480) [](ancestors = 202498480)